Makefile

```diff
@@ -1,6 +1,6 @@
 init:
-		pip install -r requirements.txt --user
+	pip install -r requirements.txt
 
 test:
-		pytest --doctest-modules --doctest-continue-on-failure --cov matchzoo/ --cov-report term-missing --cov-config .coveragerc matchzoo/ tests/
-		flake8 ./matchzoo --exclude __init__.py
+	pytest --doctest-modules --doctest-continue-on-failure --cov matchzoo/ --cov-report term-missing --cov-config .coveragerc matchzoo/ tests/
+	flake8 ./matchzoo --exclude __init__.py --ignore=D413,E126
```

Comment:
So who's making these flake8 standards? I have checks that I don't like as well.

These ignores are better stored in a `setup.cfg` file, so we can comment on what are ignored. See [here](http://flake8.pycqa.org/en/latest/user/configuration.html).

Reply:
base model can not pass flake 8 on my laptop

Reply:
solved